### PR TITLE
WHO'S YOUR FAVORITE PONY?

### PR DESCRIPTION
**Pull request by CrazyAmphibian**
## What this does
undoes the **VERY UNATOMIC CHANGE** that was in #36975

## Why it's good
it'll make sonix really mad

## How it was tested
it wasn't

## Changelog
:cl:
 * rscadd: re-added a removed beepsky drugged line.

### CODE REVIEW

**Comment by Eneocho** [comment]
image
sonix when he sees this PR (VERIFIED BY REAL AMERICAN PATRIOTS)

**Comment by SECBATON-GRIFFON** [comment]
> sonix when he sees this PR (VERIFIED BY REAL AMERICAN PATRIOTS)

it smell like cheese

**Comment by SonixApache** [comment]
repoban this lazy bastard

**Review by west3436 — APPROVED** [review]

File: code/game/machinery/bots/secbot.dm [hunk 1]
```diff
@@ -380,7 +380,7 @@ Auto Patrol: []"},
 
 /obj/machinery/bot/secbot/proc/speak(var/message)
 	visible_message("<span class='game say'><span class='name'>[src]</span> beeps, \"[message]\"",\
-		drugged_message="<span class='game say'><span class='name'>[src]</span> beeps, \"[pick("Wait! Let's be friends!","Wait for me!","You're so cool!","I-It's not like I like you or anything...","Wanna see a magic trick?","Let's go have fun, assistant-kun~")]\"")
+		drugged_message="<span class='game say'><span class='name'>[src]</span> beeps, \"[pick("Wait! Let's be friends!","Wait for me!","You're so cool!","Who's your favourite pony?","I-It's not like I like you or anything...","Wanna see a magic trick?","Let's go have fun, assistant-kun~")]\"")
```
Comment (SonixApache) [inline code/game/machinery/bots/secbot.dm:383]: horsetouchers should be repelled at all times

**Review by SonixApache — CHANGES_REQUESTED** [review]

File: code/game/machinery/bots/secbot.dm (hunk 1 above)
Comment (Eneocho) [reply code/game/machinery/bots/secbot.dm:383]: image

Comment (SECBATON-GRIFFON) [reply code/game/machinery/bots/secbot.dm:383]: bad taste in ss13 reagents tm

**Comment by west3436** [comment]
No.

**Pull request closed without merging**